<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(229, 236, 239)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <widget class="QWidget" name="inputWidget" native="true">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>108</y>
      <width>300</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>250</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>300</width>
      <height>250</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#inputWidget {
	background-color: rgb(255, 255, 255);	
	border-radius: 5;
}

QLineEdit {
	background-color: rgb(255, 255, 255);	
	border-width: 2;
	border-color: rgb(0, 0, 0);
}

QPushButton {
	border-radius: 0;
	background-color: rgb(246, 246, 246);
	color: rgb(56, 67, 79);
}	

QPushButton:hover {
	background-color: rgb(214, 214, 214)
}	
QPushButton:pressed {
	background-color: rgb(187, 187, 187)
}</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>32</number>
     </property>
     <property name="leftMargin">
      <number>64</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>64</number>
     </property>
     <property name="bottomMargin">
      <number>32</number>
     </property>
     <item>
      <widget class="QLineEdit" name="loginLineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Логин</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inputPushButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Вход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>559</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
